fix(android): update library to support 16kb

naylinndev opened the pull request.

Commit:
fix(android): update library to support 16kb

naylinndev committed:
Merge branch 'master' into support_16kb

illia-romanenko commented:
Close as https://github.com/solid-software/flutter_vlc_player/pull/539 merged. Let me know if you still miss anything.

alohadance98 commented:
> Close as #539 merged. Let me know if you still miss anything.

@illia-romanenko this pull request will fix the bug in #546

naylinndev commented:
@illia-romanenko I made a pull request #547 . In this request, I fixed the audio track and subtitle track issues on Android after updating to VLC v4.0.0, and also fixed the issue where the example app couldn’t run.

illia-romanenko commented:
Thanks @alohadance98 - can you please submit another one - I'll merge. For some reason I can't reopen this one :(
